superagi/models/models_config.py

```diff
@@ -69,7 +72,11 @@ def fetch_value_by_agent_id(cls, session, agent_id: int, model: str):
         if not config:
             return None
 
-        return {"provider": config.provider, "api_key": decrypt_data(config.api_key)} if config else None
+        if config.api_key:
```

Comment:
This fixes an issue where the API key, left blank, would create an error. Since there is no API key for users just running locally, having a blank on makes the most sense